src/IW/Sync/Search.hs

```diff
@@ -11,24 +11,31 @@ module IW.Sync.Search
        ( searchAllHaskellRepos
        , searchAllHaskellIssues
        , searchAllHaskellIssuesByLabels
-       , parseUserData
        ) where
 
 import Data.Time (Day (..), addDays)
-import GitHub (SearchResult (..), URL (..), RateLimit (..), Limits, Paths, QueryString, executeRequest', limitsRemaining)
+import GitHub ( GenRequest (..)
+              , SearchResult (..)
+              , RateLimit (..)
```

Comment:
This import is formatted in a weird way... Are you using `stylish-haskell` with the config from this repo?

Reply:
@chshersh No, I'm not using `stylish-haskell` at all. This was me. I will fix this.